geomesa-accumulo/geomesa-accumulo-tools/conf/log4j.properties

```diff
@@ -1,33 +0,0 @@
-# log to stderr to be consistent with unix (data to stdout + logging to stderr)
-log4j.rootLogger=info,file
-
-geomesa.log.dir=.
-
-# set logging levels to appropriate values
-log4j.logger.org.locationtech.geomesa.tools.accumulo.ingest=info,file
-log4j.additivity.org.locationtech.geomesa.tools.accumulo.ingest=false
```

Comment:
The root logger is configured to just print to the log file - there is a 'user' logger that should be used for things the user will see (usually validation errors, info messages, etc), and there is an 'output' logger for results. All other loggers will go to the logs.